tests/testthat/test-cnd-signal.R

```diff
@@ -92,6 +92,60 @@ test_that("`inform()` returns invisibly", {
   expect_invisible(inform("foo"))
 })
 
+test_that("warn() respects frequency", {
+  expect_warning(
```

Comment:
Should it be a public option `rlang_force_verbosity`? Or perhaps a more general `rlib_force_verbosity` that multiple packages would respond to?

Reply:
I have no strong opinions here. Except I'm not sure "verbosity" is the right word; because it's not controlling how long the output it; it's more about whether it appears every time or not. Maybe `rlib_occassional = FALSE`?

Reply:
Given the naming uncertainty, should we go with `rlang:::force_conditions = TRUE`? It's cheap to keep that options in the testthat setup even when we eventually change it when we have more confidence.

Reply:
`rlang::message_always = TRUE`? But yeah, we don't need to get it right this time.